### ISSUE TITLE
Overview of ropensci website tutorials

### ISSUE THREAD

**Issue by sckott** [ropensci-archive/intern#2]
We have tutorials on our website at https://ropensci.org/tutorials/ 

Ideally, we'd have a tutorial for each package that we have on CRAN. This is not the case right now. 

Make a list/table of ropensci packages that are on CRAN, which have tutorials and which do not. 

Then start making tutorials for packages.

**Comment by cboettig** [comment]
Sounds like said list could come from your `ropkgs` data?  I think the trick here is going to be convincing intern to do these things in a way that's programmatic and reproducible, rather than manually clicking on every repo and starting to make their own spreadsheet from scratch; even if the tutorial information is entered manually.

**Comment by sckott** [comment]
Yeah, could use `ropkgs`.  They can even scrape the tutorials page to get which pkgs have tutorials :)
